tldw_Server_API/app/api/v1/endpoints/orgs.py

```
@@ -0,0 +1,920 @@
+
+    logger.info(f"Org {ctx.org_id} updated by user with role {ctx.role}")
+    return OrganizationResponse(**org)
+
+
+@router.delete(
+    "/{org_id}",
+    status_code=status.HTTP_204_NO_CONTENT,
+    summary="Delete organization",
+)
+async def delete_org(
+    ctx: OrgContext = Depends(require_org_owner()),
```

Ruff (0.14.8)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)